app/views/steps/shared/_relationship_step.html.erb

```diff
@@ -1,17 +1,15 @@
 <% heading = t('.heading', person_name: form_object.record.person.full_name, child_name:  form_object.record.minor.full_name) %>
 
-<h1 class="heading-xlarge gv-u-heading-xxlarge">
-  <%=  heading %>
-</h1>
+<h1 class="govuk-heading-xl"><%=  heading %></h1>
 
-<%= step_form form_object do |f| %>
-  <%=
-    f.radio_button_fieldset :relation, { legend_options: { class: 'visually-hidden', text: heading } } do |fieldset|
-      fieldset.radio_input Relation::MOTHER
-      fieldset.radio_input Relation::FATHER
-      fieldset.radio_input(Relation::OTHER) { f.text_field :relation_other_value }
-    end
-  %>
+<%= step_form form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+  <%= f.govuk_radio_buttons_fieldset :relation do %>
+    <%= f.govuk_radio_button :relation, Relation::MOTHER, link_errors: true %>
+    <%= f.govuk_radio_button :relation, Relation::FATHER, link_errors: true %>
+    <%= f.govuk_radio_button :relation, Relation::OTHER do %>
+      <%= f.govuk_text_field :relation_other_value, label: { size: 'm' }, link_errors: true %>
```

Comment:
I think we've been using small size `label: { size: 's' }` for revealing elements (is that the default maybe? try removing it, if it outputs the small size, then we can go without it).